test/basic.html

```diff
@@ -73,6 +73,24 @@
           done();
         }, 12);
       });
+      
+      test('toast fires closed event', function(done) {
+        toast = fixture('basic');
+        toast.duration = 10;
```

Comment:
That sounds reasonable. But I cannot get `wct` to run for the paper-toast. For the other Polymer projects  I have here it runs just fine. But here it fails with `fixture is not a function`.